src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -106,6 +109,12 @@ class BookStep : public StepImp<TIn, TOut, BookStep<TIn, TOut>>
         return EitherAmount (cache_->out);
     }
 
+    bool
+    redeems (ReadView const& sb, bool fwd) const override
+    {
+        return !ownerPaysTransferFee_;
```

Comment:
Yes, it looks like it (I have not written a test, but reading the code). I didn't do it because with "owner pays" when the offer owner is the issuer they still don't pay a fee (in effect).

Of course, "sender pays" is a different story.

It is a good case to have a unit test for. I'll make sure I add one.